repository-data/application/src/main/resources/hcm-config/namespaces/hee/publicationLandingPage.yaml

```diff
@@ -159,16 +135,37 @@ definitions:
             hipposysedit:path: hee:featuredContentBlock
             hipposysedit:primary: false
             hipposysedit:type: hippo:mirror
+          /globalTaxonomyHealthcareTopics:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: true
+            hipposysedit:ordered: true
+            hipposysedit:path: hee:globalTaxonomyHealthcareTopics
+            hipposysedit:primary: false
+            hipposysedit:type: Taxonomy
+          /globalTaxonomyProfessions:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: true
+            hipposysedit:ordered: true
+            hipposysedit:path: hee:globalTaxonomyProfessions
+            hipposysedit:primary: false
+            hipposysedit:type: Taxonomy
+          /globalTaxonomyPublicationTypes:
```

Comment:
Could we singularise the field name ?

Reply:
My rationale for this was that I wanted to keep all the field names in line with the Taxonomy that they were reflecting. I can see that it makes sense to set this to a singular name, though, so I will change it

Reply:
Thank you. Still could see a few more occurrences on the following files. Could we fix them as well ?
- `repository-data/application/src/main/resources/hcm-config/namespaces/hee/publicationLandingPage.yaml`
- `repository-data/development/src/main/resources/hcm-content/content/documents/lks/visual-regression.yaml`
- `repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/list-item.ftl`
- `site/components/src/main/java/uk/nhs/hee/web/repository/HEEField.java`

Reply:
Will change this